keanu-project/src/main/java/io/improbable/keanu/tensor/bool/SimpleBooleanTensor.java

```diff
@@ -304,6 +304,14 @@ public int hashCode() {
         return result;
     }
 
+    @Override
+    public String toString() {
+        return "SimpleBooleanTensor{" +
+            "data=" + Arrays.toString(data) +
+            ", shape=" + Arrays.toString(shape) +
```

Comment:
Might be better to put `shape` before `data` so it's more prominent, especially when `data` is long

Reply:
Good idea! I've put shape first.